components/sandbox/toolchain-host-operator/production/stone-prd-host1/space-provisioner-configs.yaml

```diff
@@ -28,3 +28,19 @@ spec:
     maxMemoryUtilizationPercent: 90
   placementRoles:
   - cluster-role.toolchain.dev.openshift.com/tenant
+---
+apiVersion: toolchain.dev.openshift.com/v1alpha1
+kind: SpaceProvisionerConfig
+metadata:
+  name: member-kflux-prd-es01.1ion.p1.openshiftapps.com
+  namespace: toolchain-host-operator
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+spec:
+  toolchainCluster: member-kflux-prd-es01.1ion.p1.openshiftapps.com
+  enabled: true
```

Comment:
you don't want this enabled, as it will then be used when user is signed up, set to false. Since you are going to specify the member on the SpaceRequest it will pick the EaaS member only for space request which is what I think you need here

Reply:
Good to know! I've corrected it here as well as for staging.